README.md

```diff
@@ -649,8 +700,16 @@ __fastify-gql__ supports the following options:
     * `service.rewriteHeaders`: `Function` A function that gets the original headers as a parameter and returns an object containing values that should be added to the headers
     * `service.wsUrl`: The url of the websocket endpoint
     * `service.wsConnectionParams`: `Function` or `Object`
-* `persistedQueries`: A hash/query map to resolve the full query text using it's unique hash.
-* `onlyPersisted`: Boolean. Flag to control whether to allow graphql queries other than persisted. When `true`, it'll make the server reject any queries that are not present in the `persistedQueries` option above. It will also disable any ide available (playground/graphiql).
+* `persistedQueries`: A hash/query map to resolve the full query text using it's unique hash. Overrides `persistedQuerySettings`.
+* `onlyPersisted`: Boolean. Flag to control whether to allow graphql queries other than persisted. When `true`, it'll make the server reject any queries that are not present in the `persistedQueries` option above. It will also disable any ide available (playground/graphiql). Requires `persistedQueries` to be set, and overrides `persistedQuerySettings`.
+* `persistedQuerySettings`
```

Comment:
Maybe `persistedQueryStorage` is a better name? those are functions.

Reply:
Whilst most members are functions:
- `isPersistedQuery` and `getHash` functions extract data from the request payload. 
- `getQueryFromHash`, `saveQuery` are related to storage.
- `getHashForQuery` option is about hashing a query.
- `notFoundError` and `notSupportedError` are strings.

I think based on this breakdown, the options/settings are broader than "storage". Happy to change if you still would like, but maybe another name is better?

Reply:
`persistedQueryProvider` maybe?